src/Command/Status.php

```diff
@@ -33,4 +35,79 @@ protected function configure()
             ->addOption('--connection', '-c', InputArgument::OPTIONAL, 'The datasource connection to use')
             ->addOption('--source', '-s', InputArgument::OPTIONAL, 'The folder where migrations are in');
     }
+
+    /**
+     * Show the migration status.
+     *
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return void
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
```

Comment:
Because the original command has output statements in the ``Manager::printStatus()`` method.
So I moved the output in this method (because output should only be in Command classes in my opinion) to just have what was needed to get the status of migrations in the ``CakeManager::printStatus()`` method.